### PR TITLE
Add go.bin pk conversion, update deps

### PR DESCRIPTION
**Pull request by ed255**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -10,7 +10,8 @@
   },
   "license": "GPL-3.0",
   "dependencies": {
-    "circomlib": "0.2.2"
+    "circomlib": "0.2.2",
+    "wasmsnark": "0.0.10"
```
Comment (arnaucube) [inline package.json:14]: Why wasmsnark is imported in dependencies?

Comment (ed255) [reply package.json:14]: wasmsnark is required for the `buildpkey.js` utility which converts the `proving_key.json` into `proving_key.bin`.  It's used in `compile-circuit.sh`.

**Comment by ed255** [comment]
I have removed the `test/credentialDemoWrapper.test.ts` file from this PR which had wrong inputs (that's what made the tests fail) for the following reasons:
- There's already a `test/credential.test.ts` that tests the `credential.circom` circuit
- The input generation of `test/credentialDemoWrapper.test.ts` was not in `go/testvectorsgen` (I made this test to debug the credential ownership proof in the demo and took the inputs from `go-iden3-servers-demo`).

File: package.json (hunk 1 above)
Comment (arnaucube) [reply package.json:14]: but then can be moved into the devDependencies, right?
The `dependencies` is what will be installed when an external project imports this repository as a library, while the `devDependencies` is what is installed when working directly on this repository. For this repo the only `dependencies` is `circomlib`, the rest should go to `devDependencies`

Comment (ed255) [reply package.json:14]: ah, makes sense!

**Pull request merged**